### PR TITLE
add range.js 1.1.0

### LINKED ISSUE #7073: [Request] Add range.js
**Library name:** range.js
**Git repository url:** https://github.com/florian/range.js
**npm package url(optional):** range.js

### PR DESCRIPTION
**Pull request by kennynaoh**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/florian/range.js
 * Official website (optional, not the repository):
 * NPM package url (optional):range.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars:74
   - Count of watchers:4
   - Count of forks:2
 * NPM download stats (optional):
   - Downloads in the last day:1
   - Downloads in the last week:7
   - Downloads in the last month:41

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

autoupdate config added, close #7073, cc @florian

### CODE REVIEW

File: ajax/libs/range.js/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "author": {
+    "email": "jscoderr@gmail.com",
+    "name": "Florian H."
+  },
+  "dependencies": {},
+  "description": "JavaScript's missing range function.",
+  "dist": {
```
Comment (PeterDaveHello) [inline ajax/libs/range.js/package.json]: Please remove `"dist"` section.

File: ajax/libs/range.js/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "author": {
+    "email": "jscoderr@gmail.com",
+    "name": "Florian H."
+  },
+  "dependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/range.js/package.json]: Please remove empty `"dependencies"`

File: ajax/libs/range.js/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
...nrange('1..5', 2); // range(1, 5, 2)\r\n\r\n// Exclusive ranges:\r\nrange('1...5'); // [1, 2, 3, 4]\r\n```\r\n\r\n## `range.equals(rangeA, rangeB)`\r\n\r\nCompares ranges by value.\r\n\r\n```js\r\nvar a = range(1, 5);\r\nvar b = range(1, 5);\r\nvar c = range('a..z');\r\n\r\nrange.equals(a, b); // true\r\nrange.equals(a, c); // false\r\n```\r\n\r\n## `range.overlaps(rangeA, rangeB)`\r\n\r\n```js\r\nrange.overlaps(range('a..z'), range('e..f')); // true\r\nrange.overlaps(range(1, 10), range(1, 9)); // true\r\nrange.overlaps(range(1, 5), range(6, 10)); // false\r\n```\r\n\r\n- - -\r\n\r\n## Test suite\r\nThe test suite uses the [mocha](http://visionmedia.github.com/mocha) testing framework and the [chai](http://chaijs.com) assertion library.\r\n\r\nRunning the tests from the command line:\r\n\r\n```sh\r\n$ git clone git://github.com/js-coder/range.js.git && cd range.js\r\n$ npm install\r\n$ grunt test\r\n```\r\n\r\nTo run the tests in your browser, Clone this repository, and visit *./spec/index.html*.\r\n\r\n\r\n- - -\r\n\r\n## Todo\r\n\r\n- Add benchmarks\r\n- Improve performance by predefining the length\r\n- Add charCode ranges\r\n- Add `range.equals`\r\n- Add `range.overlaps`\r\n",
```
Comment (PeterDaveHello) [inline ajax/libs/range.js/package.json]: Please remove `"readme"` section.

File: ajax/libs/range.js/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "homepage": "https://github.com/js-coder/range.js#readme",
+  "license": "MIT",
+  "name": "range.js",
+  "filename": "range.min.js",
+  "npmName": "range.js",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/range.js/package.json]: It's not a good idea to use `"**/*"` in non-dist folder, please use `"range*"` instead.

**Comment by kennynaoh** [comment]
I have fixed problems mentioned above already.

File: ajax/libs/range.js/package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  "npmName": "range.js",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "range*"
+      ]
+    }
+  ],
+  "optionalDependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/range.js/package.json]: Please remove empty section.

**Comment by kennynaoh** [comment]
I have fixed the empty section in package.json already.

**Comment by PeterDaveHello** [comment]
Looks better, please rebase your branch with our latest master branch, thanks.

**Comment by kennynaoh** [comment]
I haved rebased my branch, thanks

**Comment by PeterDaveHello** [comment]
Nice, please also try to add some keywords next time.

**Pull request merged**